pygotham/frontend/talks.py

```diff
@@ -100,6 +100,13 @@ def proposal(pk=None):
         db.session.add(talk)
         db.session.commit()
 
+        speaker = Speaker(
```

Comment:
Are we planning to let all co-presenters edit the talk? We'll need to figure out how we handle two people editing the same talk at the same time.